### ISSUE TITLE
Thank You Thread

### ISSUE THREAD

**Issue by xpe** [alangrainger/obsidian-frontmatter-modified-date#38]
@alangrainger Thanks for making this!

**Comment by alangrainger** [comment]
I appreciate that :) I've been closing recent "issues" and they are all things solved by looking in the Settings page.

It's exhausting doing this over multiple plugins 😩

**Comment by GlenPerkins** [comment]
@alangrainger I especially want to thank you for (1) writing the actual text values into the note instead of just a template placeholder. That means the auto-updated data will be readable outside of Obsidian, which I thought everyone would insist on. And for (2), basing the last updated timestamp on the clock's "now" instead of the file's "modified". That seems obvious, too. When files are passed around, creation and modified dates get reset, so that's not what I mean. I mean the last time the *content* was changed.

I went looking for both seemingly obvious requirements. No one seemed to be thinking this way until I found your plugin, so definitely thanks.

**Comment by hashhar** [comment]
Indeed, this is one of the plugins that does it's job so well that I never noticed it's there.

I recently set up a new vault and very quickly realised I was missing this plugin there. Indispensible tool and the only one with a sane behaviour as said by @GlenPerkins.
